docs/content/doc/packages/cran.en-us.md

```diff
@@ -0,0 +1,96 @@
+---
+date: "2023-01-01T00:00:00+00:00"
+title: "CRAN Packages Repository"
+slug: "packages/cran"
+draft: false
+toc: false
+menu:
+  sidebar:
+    parent: "packages"
+    name: "CRAN"
+    weight: 35
+    identifier: "cran"
+---
+
+# CRAN Packages Repository
+
+Publish [CRAN](https://cran.r-project.org/) packages for your user or organization.
+
+**Table of Contents**
+
+{{< toc >}}
+
+## Requirements
+
+To work with the CRAN package registry, you need to install [the R toolset](https://cran.r-project.org/).
+
+## Configuring the package registry
+
+To register the package registry you need to add it in your `Rprofile.site` file:
+
+```
+local({
+	r <- list("gitea" = "https://gitea.example.com/api/packages/{owner}/cran")
+	options(repos = r)
+})
```

Comment:
This is somewhat dangerous to apply as it will override the existing `"repos"` option.
It's better to append to the existing ones, to avoid breaking other configured repos.

```
options("repos" = c(getOption("repos"), c(gitea="https://gitea.example.com/api/packages/{owner}/cran")))
```

Also the `local()` call is not needed here.